src/Dhall/Pretty.hs

```diff
@@ -0,0 +1,3 @@
+module Dhall.Pretty (prettyExpr) where
+
+import Dhall.Pretty.Internal
```

Comment:
What's the reason for the separate `Dhall.Pretty.Internal` module?  Is this related to the use of mutually recursive modules?

Reply:
As mentioned in the PR description:

> Dhall.Core actually relies on some "internal" pretty printing functions,
so pretty printing has been moved to Dhall.Pretty.Internal - Dhall.Pretty
exposes the public facing interface.

Does that help? It's things like `Core` needing `buildConst` (see the `Internal.hs-boot` for exactly what `Core` requires).

Reply:
Oh, whoops!  I missed that.  That makes sense